faulthandler.c

```diff
@@ -967,28 +1019,34 @@ Py_uintptr_t
 stack_overflow(Py_uintptr_t min_sp, Py_uintptr_t max_sp, size_t *depth)
 {
     /* allocate 4096 bytes on the stack at each call */
-    unsigned char buffer[4096];
+    unsigned char buffer[1024*1024];
     Py_uintptr_t sp = (Py_uintptr_t)&buffer;
     *depth += 1;
-    if (sp < min_sp || max_sp < sp)
+    if (sp < min_sp || max_sp < sp) {
+        printf("call #%lu\n", (unsigned long)*depth);
```

Comment:
Please remove the printf.

Reply:
Removed it by amending the commit so as to keep the pull request clean